README.md

```diff
@@ -33,29 +33,44 @@ Authorization code provider is used by Web Apps (ASP.NET & ASP.NET Core) to acqu
 It uses [MSALs Authorization Code](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Acquiring-tokens-with-authorization-codes-on-web-apps) to authenticate Microsoft Graph requests.
 
 ```csharp
-IConfidentialClientApplication clientApplication = AuthorizationCodeProvider.CreateClientApplication(clientId, redirectUri, clientCredential);
-
-AuthorizationCodeProvider authenticationProvider = new AuthorizationCodeProvider(clientApplication, scopes);
+IConfidentialClientApplication confidentialClientApplication = ConfidentialClientApplicationBuilder
+                .Create(clientId)
+                .WithRedirectUri(redirectUri)
+                .WithClientSecret(clientSecret)
+                .WithAuthority(authority)
```

Comment:
Is it necessary to include .WithAuthority()?   I'd rather not include optional and unneeded properties in these examples.

Reply:
I wanted to show the configuration options available, but MSAL's docs already do that. Let me keep it simple and remove it.